core/store/orm/orm.go

```diff
@@ -1294,15 +1294,29 @@ func (orm *ORM) GetDefaultAddress() (common.Address, error) {
 
 // NOTE: We can add more advanced logic here later such as sorting by priority
 // etc
-func (orm *ORM) getDefaultKey() (models.Key, error) {
-	availableKeys, err := orm.Keys()
-	if err != nil {
+func (orm *ORM) getRoundRobinKey(id models.ID) (models.Key, error) {
+	idNum := 0
+	for _, b := range id.Bytes() {
+		idNum += int(b)
+	}
```

Comment:
Can you please explain this code? What are you trying to achieve here?

Reply:
This was just an attempt at getting an int-representation of the job run id, so we could use that in modulo with number of available keys. Though, we decided to go with another approach now.

The reason for picking a random key and not the first key is that some customers prefer balancing the load on different addresses due to throughput concerns.